.pylintdict

```diff
@@ -306,6 +306,7 @@ implicant
 implicants
 indvar
 init
+initialise
```

Comment:
This should not be needed if whatever is spelled that way is changed to `initialize` i.e. the US spelling (rather than s that is used in other variants). Ideally this should be restricted to US English spelling

Reply:
Removed and changed the comments.